src/PKSim.Core/CoreConstants.cs

```diff
@@ -26,8 +29,7 @@ public static class CoreConstants
       public const double HIGH_RESOLUTION_IN_PTS_PER_MIN = 1.0 / 3; //20 pts per hours
       public const double LOW_RESOLUTION_IN_PTS_PER_MIN = 1.0 / 15; //4 pts per hours
       public const double DEFAULT_PROTOCOL_END_TIME_IN_MIN = 1440;
-      public const byte DEFAULT_ONTOGENY_FACTOR = 1;
```

Comment:
Changes in this file are simple reordering and removing of old constants not used anymore. There are always some that we miss :)